pkg/ccl/backupccl/backup_test.go

```diff
@@ -8236,21 +8244,11 @@ func TestFullClusterTemporaryBackupAndRestore(t *testing.T) {
 	defer cleanupRestore()
 	sqlDBRestore.Exec(t, `RESTORE FROM 'nodelocal://0/full_cluster_backup'`)
 
-	// Before the reconciliation job runs we should be able to see the following:
-	// - 4 synthesized pg_temp sessions in defaultdb.
-	// We synthesize a new temp schema for each unique backed-up <dbID, schemaID>
-	// tuple of a temporary table descriptor.
-	// - All temp tables remapped to belong to the associated synthesized temp
-	// schema, and in the defaultdb.
-	checkSchemasQuery := `SELECT schema_name FROM [SHOW SCHEMAS] WHERE schema_name LIKE 'pg_temp_%' ORDER BY
-schema_name`
-	sqlDBRestore.CheckQueryResults(t, checkSchemasQuery,
```

Comment:
I changed it to count since we're not guaranteed ordering for the numbers.

Ie `pg_temp_0_1` can be in any of the dbs. Does the numbering matter in this case?